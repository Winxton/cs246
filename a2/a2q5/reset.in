ra
1
2
3
100
1
2
3
x
pa
ra
1
2
3
x
pa
ra
1
2
3
4
5
6
7
8
9
10
11
x
pa
q
