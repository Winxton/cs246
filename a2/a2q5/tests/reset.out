1 2 3 100 1 2 3 
Capacity: 10
1 2 3 
Capacity: 5
1 2 3 4 5 6 7 8 9 10 11 
Capacity: 20
